### DISCUSSION TITLE
Github only show the title of my website

### DISCUSSION THREAD

**Discussion by fretagi** [community/community#25328] [Actions]
<p>I am new to github. I recentley uploaded the files for my first website, and in settings its says that the page can be viewed, but it only shows the title.</p>

**Comment by madebyjeffrey** [comment]
<p>Are you talking about <a href="https://github.com/fretagi/MyFirstWebsite" rel="nofollow noopener">this repository</a>?</p>
<p>According to <a href="https://pages.github.com/" rel="nofollow noopener">the Github Pages</a> you must name it as fretagi . github . io.</p>

**Comment by fretagi** [comment, marked as answer]
<p>What I did was actually rename the file receita.html to index.html, although the pics are not shown in the website…</p>

**Comment by taposwini** [comment]
https://taposwini.github.io/dr-maheswari/
yes same problem.pls help

**Comment by Rohan13253** [comment]
I am also facing same issue in which only website heading is showing and the file is not in html it is in python i tried to change its name to index.py but nothing changed

**Reply by wrightedgar47-code** [reply]
same to me

**Comment by liljayjay247** [comment]
same thing is going on and its my school finals
